lib/LaTeXML/Core/Document.pm

```diff
@@ -1419,8 +1419,20 @@ sub addSSValues {
   return; }
 
 sub addClass {
-  my ($self, $node, $class) = @_;
-  return addSSValues($self, $node, class => $class); }
+  my ($self, $node, $value) = @_;
+  return addSSValues($self, $node, class => $value); }
+
+sub removeClass {
```

Comment:
May be useful in the future to generalize this to `removeSSValues` and add a layer like `addClass`

Reply:
Done, thanks!